tests/Integration/Config/ComponentProvider/Metrics/MetricExporterPrometheus.php

```diff
@@ -70,6 +70,7 @@ public function getConfig(ComponentProviderRegistry $registry, NodeBuilder $buil
                         ->end()
                     ->end()
                 ->end()
+                ->scalarNode('translation_strategy')->defaultValue('UnderscoreEscapingWithSuffixes')->end()
```

Comment:
Should remove `without_units` and `without_type_suffix` as these two options were replaced by `translation_strategy`.

```suggestion
                ->enumNode('translation_strategy')
                    ->defaultValue('UnderscoreEscapingWithSuffixes')
                    ->values([
                        'UnderscoreEscapingWithSuffixes',
                        'UnderscoreEscapingWithoutSuffixes',
                        'NoUTF8EscapingWithSuffixes',
                        'NoTranslation',
                    ])
                ->end()
```